### PR TITLE
[MRG] Notation

### PR DESCRIPTION
**Pull request by xiaotinghe**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-559/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-559/

File: chapter_notation/index.md [hunk 1]
```diff
@@ -1,73 +1,75 @@
 # 符号
 :label:`chap_notation`
 
-本书中使用的符号概述如下。
+本书中使用的符号概述如下:
```
Comment (astonzhang) [inline chapter_notation/index.md:4]: 句号

File: chapter_notation/index.md [hunk 2]
```diff
@@ -1,73 +1,75 @@
...
 
 ## 数字
```
Comment (astonzhang) [inline chapter_notation/index.md:7]: 把“一个”都删了。这里的冠词无实际意义，不需要翻译

File: chapter_notation/index.md [hunk 3]
```diff
@@ -1,73 +1,75 @@
...
-* 第 $i$ 行和第 $j$ 栏中的矩阵元素：第 $\mathbf{X}$ 行和第 $j$ 栏中的矩阵元素
+* $\mathbf{I}$：单位矩阵
+* $x_i$, $[\mathbf{x}]_i$：向量$\mathbf{x}的第$i^\mathrm{th}$个元素
+* $x_{ij}$, $[\mathbf{X}]_{ij}$：矩阵$\mathbf{X}$第$i$行第$j$列的元素
 
-## 集理论
+## 集合
 
-* 一套
+* $\mathcal{X}$: 一个集合
```
Comment (astonzhang) [inline chapter_notation/index.md:18]: 集合

File: chapter_notation/index.md [hunk 4]
```diff
@@ -1,73 +1,75 @@
...
+* $\mathbf{X}$：一个矩阵
 * $\mathsf{X}$：一个张量
-* $\mathbf{I}$：身份信息矩阵
-* 传播媒介的元素：传播媒介的元素
-* 第 $i$ 行和第 $j$ 栏中的矩阵元素：第 $\mathbf{X}$ 行和第 $j$ 栏中的矩阵元素
+* $\mathbf{I}$：单位矩阵
+* $x_i$, $[\mathbf{x}]_i$：向量$\mathbf{x}的第$i^\mathrm{th}$个元素
+* $x_{ij}$, $[\mathbf{X}]_{ij}$：矩阵$\mathbf{X}$第$i$行第$j$列的元素
 
-## 集理论
```
Comment (astonzhang) [inline chapter_notation/index.md:16]: 集合论

File: chapter_notation/index.md [hunk 5]
```diff
@@ -1,73 +1,75 @@
...
-* $\odot$: 哈达玛（元素）产品
-* $[\cdot, \cdot]$：连接
-* $\lvert \mathcal{X} \rvert$：一套集合的基数
-* 标准：标准
-* 标准：标准
-* $\langle \mathbf{x}, \mathbf{y} \rangle$：载体的点积和点积
-* $\sum$: 系列增加
-* $\prod$: 串联乘法
+* $\mathbf{X}^{-1}$: 矩阵的逆
+* $\odot$: Hadamard乘积（逐元素乘）
```
Comment (astonzhang) [inline chapter_notation/index.md:35]: 按元素相乘

**Comment by mli** [comment]
Job d2l-zh/PR-559/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-559/

**Comment by mli** [comment]
Job d2l-zh/PR-559/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-559/

**Comment by mli** [comment]
Job d2l-zh/PR-559/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-559/

**Pull request merged**